packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -211,26 +173,4 @@ export class EVMAAWallet extends LoggerWrapper {
             return this.crossmintService.fetchNFTs(this.account.address, this.chain);
         });
     }
-
-    private getLegacyTransactionFeesParamsIfApply(gasFeeParams?: {
-        maxFeePerGas?: BigNumberish;
-        maxPriorityFeePerGas?: BigNumberish;
-    }) {
-        const { maxFeePerGas, maxPriorityFeePerGas } = gasFeeParams ?? {};
-
-        if (hasEIP1559Support(this.chain)) {
-            return {
-                // only include if non-null and non-zero
-                ...(maxFeePerGas && { maxFeePerGas: BigInt(maxFeePerGas.toString()) }),
-                ...(maxPriorityFeePerGas && { maxPriorityFeePerGas: BigInt(maxPriorityFeePerGas.toString()) }),
-            };
-        } else {
-            if (maxFeePerGas || maxPriorityFeePerGas) {
-                console.warn(
-                    "maxFeePerGas and maxPriorityFeePerGas are not supported on this chain as it supports Legacy Transacitons. Ignoring them."
-                );
-            }
-            return gelatoBundlerProperties;
-        }
-    }
```

Comment:
I forget if Matias or Maxi added this, there was some issue when we send transactions to the gelato bundler where we needed this
This is forced when we use astar because entry point v6 is the only one currently supported

Reply:
`smartAccount.ts` has a replacement for this!